css/BEM-style.md

```diff
@@ -17,11 +17,11 @@ About 30 - 45 minutes
   
 ### Motivation
 
-On smaller pet projects, the way you organise your CSS or even SaSS styles doesn't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time.
+On smaller pet projects, the way you organise your CSS or even Sass styles doesn't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time.
 
-However in bigger projects with over a 100 stylesheets and varied components spread across the application, that can raise a serious issue, since changing the styles for any compponent would mean searching for the absurd CSS naming (either id's, classes or combination of both ) you provided maybe a month before.
+However in bigger projects with over a 100 style sheets and varied components spread across the application, that can raise a serious issue, since changing the styles for any component would mean searching for the absurd CSS naming (either id's, classes or combination of both) you provided maybe a month before.
 
-If that's not worse, how about you also finding out that your new styles written lack behind on specificity and your styles were never applied, since some other previously applied class or even id made your previous styles highly specific compared to your new ones.
+If that's not worse, how about you also finding out that your new styles written lacks behind on specificity and your styles were never applied, since some other previously applied class or even id made your previous styles highly specific compared to your new ones.
```

Comment:
```suggestion
If that's not worse, how about you also finding out that your new styles written lags behind on specificity and your styles were never applied, since some other previously applied class or even id made your previous styles highly specific compared to your new ones.
```
?

Would you mind re-wording this whole section?  It sounds a but funny.